tests-e2e/cypress/integration/adminconsole/analytics_spec.js

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// ***************************************************************
+
+describe('admin console > site statistics', () => {
+    let testUser;
+    let testTeam;
+    let testPlaybook;
+    let testSysadmin;
+
+    before(() => {
+        cy.apiInitSetup().then(({team, user}) => {
+            testTeam = team;
+            testUser = user;
+            cy.apiCreateCustomAdmin().then(({sysadmin}) => {
+                testSysadmin = sysadmin;
+            });
+            cy.apiCreatePlaybook({
+                teamId: testTeam.id,
+                title: 'Playbook',
+                memberIDs: [],
+            }).then((playbook) => {
+                testPlaybook = playbook;
+            });
+        });
+    });
+
+    describe('site statistics', () => {
+        it('playbooks and runs counters are visible', () => {
+            cy.apiLogin(testSysadmin);
+
+            // # Go to admin console > site statistics
+            cy.visit('/admin_console/reporting/system_analytics');
+            cy.findByTestId('playbooks.playbook_count').should('exist');
+            cy.findByTestId('playbooks.playbook_run_count').should('exist');
```

Comment:
Can we add a validation comment here? Something like
```suggestion

            // * Check that the playbook and run counters are visible
            cy.findByTestId('playbooks.playbook_count').should('exist');
            cy.findByTestId('playbooks.playbook_run_count').should('exist');
```